alembic/versions/d28e52d86633_add_username_to_manager.py

```diff
@@ -0,0 +1,28 @@
+"""Add username to manager
+
+Revision ID: d28e52d86633
+Revises: 
+Create Date: 2019-07-02 22:51:25.613742
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd28e52d86633'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('queue_manager', sa.Column('username', sa.String(), nullable=True))
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('queue_manager', 'username')
+    # ### end Alembic commands ###
```

Comment:
Can we move this inside the `qcfractal/*` folder so that users can upgrade their database after they have installed the project?